src/metabase/lib/common.cljc

```diff
@@ -28,8 +28,17 @@
   :hierarchy lib.hierarchy/hierarchy)
 
 (defmethod ->op-arg :default
-  [_query _stage-number x]
-  x)
+  [query stage-number x]
+  (if (and (vector? x)
+           (keyword? (first x)))
```

Comment:
Might reuse `metabase.lib.options/mbql-clause?`. There are a number of places where we do this test.